### DISCUSSION TITLE
How to finetune on my dataset?

### DISCUSSION THREAD

**Discussion by songgaojundeng** [SalesforceAIResearch/uni2ts#80] [General]
Thank you very much for your work and your open-source code. The description for fine-tuning on customized datasets is unclear to me.

I firstly prepared and saved the dataset based on the [notebook](https://github.com/SalesforceAIResearch/uni2ts/blob/main/example/prepare_data.ipynb) provided by the authors. In **Finetune** 2. I don't know how to process the dataset into the required format, e.g., the data I generated are folders including an arrow file. 

The example provided by the authors is:
```
python -m cli.train \
  -cp conf/finetune \
  run_name=example_run \ 
  model=moirai_1.0_R_small \ 
  data=etth1 \ 
  val_data=etth1  
```
I'm also not sure how to set up the configuration files (.yaml) and where to put them, etc. Can you update the README with clearer instructions?

**Comment by songgaojundeng** [comment]
I made it work by guessing and checking the code.

**Comment by gorold** [comment]
that's great! I've also replied on the issue you opened. we can discuss there further if needed.
